include/BMS.hpp

```diff
@@ -337,21 +386,19 @@ class BMS {
 
         // TPDO0 settings
         // 0: The TPDO number, default 0
-        // 1: The COB-ID used by TPDO0, provided as a function of the TPDO
-        //    number
-        // 2: How the TPDO is triggered, defaulting to manual trigger
+        // 1: The COB-ID used by TPDO0, provided as a function of the TPDO number
+        // 2: How the TPO is triggered, default to manual triggering
         // 3: Inhibit time, defaults to 0
-        // 5: Timer trigger time in 1ms units, 0 will disable the timer based
-        //    triggering
+        // 5: Timer trigger time in 1ms units, 0 will disable the timer based triggering
```

Comment:
Should be 3, 4 not 3, 5. this seems to be an issue throughout most of the docs for the TPDOs